test/integration/resize_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("vary VM parameters: memory cpus, disk", func() {
+var _ = Describe("vary VM parameters: memory cpus, disk", Label("openshift-preset"), func() {
```

Comment:
I'd use `disk-resize` as the label here, even though this performs the test with `openshift` preset

Reply:
I added `disk-resize` label (unused as of now). You seem to be assuming we can only have 1 label. Since we can have many, do you still want to get rid of the `openshift-preset` label? I wouldn't be in favor of that.

Reply:
yes right, assumed we can only add one label.. having both is better, thanks